src/OpenColorIO/Baker.cpp

```diff
@@ -220,9 +241,156 @@ void Baker::bake(std::ostream & os) const
         throw Exception(err.str().c_str());
     }
 
+    FormatInfoVec fmtInfoVec;
+    fmt->getFormatInfo(fmtInfoVec);
+    FormatInfo fmtInfo = fmtInfoVec[0];
+
+    const std::string & inputSpace = getImpl()->m_inputSpace;
+    const std::string & targetSpace = getImpl()->m_targetSpace;
+    const std::string & shaperSpace = getImpl()->m_shaperSpace;
+    const std::string & display = getImpl()->m_display;
+    const std::string & view = getImpl()->m_view;
+    const std::string & looks = getImpl()->m_looks;
+
+    const bool displayViewMode = !display.empty() && !view.empty();
+    const bool colorSpaceMode = !targetSpace.empty();
+
+    // Settings validation.
     if(!getConfig())
     {
-        throw Exception("No OCIO config has been set");
+        throw Exception("No OCIO config has been set.");
+    }
+
+    if(inputSpace.empty())
+    {
+        throw Exception("No input space has been set.");
+    }
+
+    if(!displayViewMode && !colorSpaceMode)
+    {
+        throw Exception("No display / view or target colorspace has been set.");
+    }
+
+    if(displayViewMode && colorSpaceMode)
+    {
+        throw Exception("Cannot use both display / view and target colorspace.");
+    }
+
+    if(displayViewMode && !looks.empty())
+    {
+        throw Exception("Cannot use looks with display / view.");
```

Comment:
...no? I'd have expected looks to work with display / view. 

(presumably, any look expressions would be prepended to any existing looks in the processor; but, conceivably, one might want to bypass existing looks, and replace with user-provided.)

Example use case: creating variants of a DisplayView transform, both with and without the ACES-1.3 Reference Gamut Compression LMT.

Reply:
Not sure what are we expecting here, `DisplayViewTransform` doesn't allow for looks override either. I would assume your use case can be addressed by just having dedicated views with ACES RGC on?

If we want to still allow user override of looks, it would mean using a `LookTransform` followed by a `DisplayViewTransform` with the `setLooksBypass(true)`? Allowing prepending or appending of looks to the existing looks seems a bit complex to me.

What do you think?

Reply:
Oh, you know what? I'm getting mixed up with some of the `LegacyViewingPipeline` behavior, which is definitely overkill here. Forget I said anything...!

Reply:
I think you have a good point, we might at least support looks and use that as overrides.

Reply:
Added support for looks overrides.